testdata/jennies/rawtypes/struct_with_optional_fields.txtar

```diff
@@ -254,6 +261,24 @@ class SomeStruct:
             payload["FieldAnonymousStruct"] = self.field_anonymous_struct.to_json()
         return payload
 
+    @classmethod
+    def from_json(cls, data: dict[str, typing.Any]) -> typing.Self:
+        args = {
```

Comment:
Should we format with a tool like "black"? That would remove weird new spaces like this and probably make the generation slightly easier (less important to have correct spacing)

Reply:
As an optional post-processing step, maybe?

My initial gut feeling is that requiring third-party tools to be installed to run cog is going to be painful (especially as we add more languages)